README.md

```diff
@@ -123,12 +140,12 @@ If you're using [server-only integration](https://github.com/UnifiedID2/uid2docs
 
 ## Usage for UID Sharers
 
-A UID Sharer is a participant that wants to share UID2s or EUIDs with another participant. Raw UIDs must be encrypted into UID tokens before sending them to another participant. See com.uid2.client.test.IntegrationExamples (runSharingExample method) for an example usage.
+A UID2 Sharer is a participant that wants to share UID2s or EUIDs with another participant. Raw UIDs must be encrypted into UID tokens before sending them to another participant. See com.uid2.client.test.IntegrationExamples (runSharingExample method) for an example usage. **(gwh/JN_09 can we link to this?)**
```

Comment:
Yes, as above it's https://github.com/IABTechLab/uid2-client-java/blob/master/src/test/java/com/uid2/client/test/IntegrationExamples.java

Reply:
Done, thx.